test/k8sT/Services.go

```diff
@@ -205,81 +205,45 @@ var _ = Describe("K8sValidatedServicesTest", func() {
 	Context("External services", func() {
 
 		var (
-			endpointPath      string
-			expectedCIDR      = "198.49.23.144/32"
-			podName           = "toservices"
-			podPath           string
-			policyPath        string
-			policyLabeledPath string
-			servicePath       string
+			expectedCIDR = "198.49.23.144/32"
+			podName      = "toservices"
+
+			endpointPath      = helpers.ManifestGet("external_endpoint.yaml")
+			podPath           = helpers.ManifestGet("external_pod.yaml")
+			policyPath        = helpers.ManifestGet("external_policy.yaml")
+			policyLabeledPath = helpers.ManifestGet("external_policy_labeled.yaml")
+			servicePath       = helpers.ManifestGet("external_service.yaml")
 		)
 
 		BeforeEach(func() {
-			servicePath = helpers.ManifestGet("external_service.yaml")
-			res := kubectl.Apply(servicePath)
-			res.ExpectSuccess(res.GetDebugMessage())
-
-			endpointPath = helpers.ManifestGet("external_endpoint.yaml")
-			podPath = helpers.ManifestGet("external_pod.yaml")
-			policyPath = helpers.ManifestGet("external_policy.yaml")
-			policyLabeledPath = helpers.ManifestGet("external_policy_labeled.yaml")
+			kubectl.Apply(servicePath).ExpectSuccess("cannot install external service")
```

Comment:
IMO i think we should still error check any `Apply` statements that fail in before each.

Reply:
Edit: ignore this, I am wrong :)